web/wp-content/plugins/tmbi-salesforce-dmp/js/krux-click-traking.js

```diff
@@ -0,0 +1,28 @@
+jQuery(document).ready(function($) {
```

Comment:
This file should be part of the theme, as the markup is decided from the theme layer. E.g. an element with class `gform_wrapper` containing a child element with class `newsletter-signup` - If we keep it here, when we alter the theme's CSS or HTML structure we'll need to rewrite this part of the plugin.